backend/api/views.py

```diff
@@ -107,32 +116,105 @@ def create_job(self, request):
             raise exceptions.NotFound("Selected printer does not exist")
         job = self._create_printing_job(printer_with_perms=printer_with_perms, **serializer.validated_data)
         return Response(self.get_serializer(job).data)
+    
+    @action(detail=True, methods=['post'], name='Change job properties')
+    def change_properties(self, request, pk=None):
```

Comment:
Document the semantics of this function, most notably the fact that it requires all fields to be provided or they will be set to their default values.